package.json

```diff
@@ -34,18 +33,15 @@
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
     "system-test": "mocha build/system-test --timeout 600000",
     "cleanup": "mocha scripts/cleanup.js --timeout 30000",
-    "test": "c8 mocha build/test build/test/common",
+    "test": "mocha build/test build/test/common",
```

Comment:
What happened to `c8`?

Reply:
Putting it back! It spammed my terminal window and got punished.